### PR TITLE
Adding BERT model to fetch-data model in split ONNX format

### PR DESCRIPTION
**Pull request by lperryp**
## Status
Ready/In Progress/In Hold (Reason for hold)

## Related Content Pull Request
Related PR: link to the PR at demisto/content

## Related Issues
Related: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

### CODE REVIEW

**Comment by glicht** [comment]
@lperryp the image size is very large. I think we can reduce this if we use a "build container to unzip and do the merge". See how we do this at yolo-coco: https://github.com/demisto/dockerfiles/blob/master/docker/yolo-coco/Dockerfile#L1 the line: `FROM alpine:3.12.3 as files-src` is used for a build image were we merge the files and only then copy the merged file using: `COPY --from=files-src /yolo-coco/yolov3.weights /yolo-coco/.`

**Comment by lperryp** [comment]
@glicht thanks it reduced the size! seems fine?

File: docker/fetch-data/Dockerfile [hunk 1]
```diff
@@ -20,4 +28,9 @@ COPY ./domain_to_rank.p /ml/domain_to_rank.p
 COPY ./word_to_ngram.p /ml/word_to_ngram.p
 COPY ./word_to_regex.p /ml/word_to_regex.p
 
+COPY --from=files-src /distilbert-base-uncased.onnx /ml/distilbert-base-uncased.onnx
+
+COPY ./distilled_bert_base_uncased_tokenizer.zip /ml/distilled_bert_base_uncased_tokenizer.zip
+RUN apt-get install unzip && unzip /ml/distilled_bert_base_uncased_tokenizer.zip -d /ml && rm /ml/distilled_bert_base_uncased_tokenizer.zip
```
Comment (glicht) [inline docker/fetch-data/Dockerfile:34]: @lperryp I think we can reduce the image size if we do the unzip inside the build container and copy the unzipped directory.

**Review by glicht — APPROVED** [review]
See my changes in the latest commit. Image is smaller now.

**Comment by lperryp** [comment]
> See my changes in the latest commit. Image is smaller now.

great, thanks a lot!

**Pull request merged**